administrator/components/com_actionlogs/src/Model/ActionlogsModel.php

```diff
@@ -107,8 +117,9 @@ protected function getListQuery()
 
         // Apply filter by user
         if (!empty($user)) {
-            $user = (int) $user;
-            $query->where($db->quoteName('a.user_id') . ' = :userid')
+            $user = (int)$user;
```

Comment:
```suggestion
            $user = (int) $user;
```
IDE side-effect or c/s moving forward?

Reply:
IDE, I will check if I find the switch for it (there are many)

Reply:
found it